### PR TITLE
Added hosted service to sync academy conversion projects

### PR DESCRIPTION
**Pull request by madeviv**
- Hosted service to sync a2b projects
- Integration tests

### CODE REVIEW

File: TramsDataApi.ApplyToBecome/ApplyToBecomeDbContext.cs [hunk 1]
```diff
@@ -1,14 +1,25 @@
 ﻿using Microsoft.EntityFrameworkCore;
 
-namespace TramsDataApi.DatabaseModels
+namespace TramsDataApi.ApplyToBecome
 {
-    public partial class LegacyTramsDbContext
```
Comment (madeviv) [inline TramsDataApi.ApplyToBecome/ApplyToBecomeDbContext.cs:5]: Note sure why it thinks this is a renamed file this shouldn't be a compare

File: TramsDataApi.ApplyToBecome/SyncIfdPipeline.cs [hunk 2]
```diff
@@ -7,10 +7,10 @@
 // If you have enabled NRTs for your project, then un-comment the following line:
 // #nullable disable
 
-namespace TramsDataApi.DatabaseModels
+namespace TramsDataApi.ApplyToBecome
 {
     [Table("IfdPipeline", Schema = "mstr")]
-    public partial class IfdPipeline
```
Comment (madeviv) [inline TramsDataApi.ApplyToBecome/SyncIfdPipeline.cs:13]: Same again not a renamed file just deleted the one that is no longer needed

File: TramsDataApi.Test/Integration/AcademyConversionProjectsIntegrationTests.cs [hunk 3]
```diff
@@ -167,47 +123,46 @@ public async Task Patch_request_should_be_a_not_found_response_when_id_does_not_
...
-            academyConversionProject.SchoolBudgetInformationAdditionalInformation.Should().Be(updateRequest.SchoolBudgetInformationAdditionalInformation);
-            academyConversionProject.SchoolBudgetInformationSectionComplete.Should().Be(updateRequest.SchoolBudgetInformationSectionComplete);
+            academyConversionProject.RevenueCarryForwardAtEndMarchCurrentYear.Should().Be(updateRequest.RevenueCarryForwardAtEndMarchCurrentYear);
+            academyConversionProject.ProjectedRevenueBalanceAtEndMarchNextYear.Should().Be(updateRequest.ProjectedRevenueBalanceAtEndMarchNextYear);
+            academyConversionProject.CapitalCarryForwardAtEndMarchCurrentYear = updateRequest.CapitalCarryForwardAtEndMarchCurrentYear;
+            academyConversionProject.CapitalCarryForwardAtEndMarchNextYear = updateRequest.CapitalCarryForwardAtEndMarchNextYear;
+            academyConversionProject.SchoolBudgetInformationAdditionalInformation = updateRequest.SchoolBudgetInformationAdditionalInformation;
+            academyConversionProject.SchoolBudgetInformationSectionComplete = updateRequest.SchoolBudgetInformationSectionComplete;
```
Comment (rebeccafitzsimmons1) [inline TramsDataApi.Test/Integration/AcademyConversionProjectsIntegrationTests.cs:150]: is this change right?

**Review by rebeccafitzsimmons1 — APPROVED** [review]
nice one

**Review by RobertM905 — APPROVED** [review]
Looks good 👍

**Pull request merged**